test/intlinear.jl

```diff
@@ -0,0 +1,70 @@
+function set_knapsacktest_solutions!(solver)
+    # SOLVE 1
+    LQOI.set_solution!(solver,
+    termination_status = MOI.Success,

```

Comment:
See comment about sensible defaults, then we can remove things like this.

(Also Style: indentation.)